History.md

```diff
@@ -1,3 +1,5 @@
+* BMurri: WIXBUG:4532 - Make it an error for InstallPrivileges and InstallScope to specify contradictory values.
```

Comment:
That's a great idea. I'll update my pull requests to align with this within 24 hours or so.